src/requesthandler/RequestHandler.cpp

```diff
@@ -166,6 +166,7 @@ const std::unordered_map<std::string, RequestMethodHandler> RequestHandler::_han
 	{"StartStream", &RequestHandler::StartStream},
 	{"StopStream", &RequestHandler::StopStream},
 	{"SendStreamCaption", &RequestHandler::SendStreamCaption},
+	{"GetStreamScreenshot", &RequestHandler::GetStreamScreenshot},
```

Comment:
I agree, the `GetProgramScreenshot` seems like a better name since "Program" is also used in the user-facing part. But I wasn't sure about the naming conventions on obs-websocket, so yes, we may need @tt2468 feedback on this :)

My first idea was `GetOutputScreenshot`, but that was too generic. Then I renamed it to `GetStreamScreenshot`.